pkgs/applications/emulators/duckstation/default.nix

```diff
@@ -61,9 +78,9 @@ stdenv.mkDerivation {
   ++ cubeb.passthru.backendLibs;
 
   cmakeFlags = [
-    "-DUSE_DRMKMS=ON"
```

Comment:
If I'm not mistaken, with that removed, `mesa` can be removed from the dependencies.

Reply:
duckstation searches for EGL.

Reply:
I tried removing `mesa`, EGL is found anyway:
```
-- Found EGL: /nix/store/8gp0sr5qn61f17z8vza7nm32kshbj679-libGL-1.6.0-dev/include (found version "1.5") 
```

Reply:
How precisely? libGL is not an explicit input.

Reply:
It's from https://github.com/NixOS/nixpkgs/blob/c0d8c84f4024a3b2b6a7b648a3b692532bd84a2c/pkgs/development/libraries/qt-6/modules/qtbase.nix#L173

Reply:
Nice.